modules/blazingsql/src/blazingcontext.ts

```diff
@@ -42,7 +43,7 @@ export class BlazingContext {
   private generator: any;
   private tables: Map<string, DataFrame>;
 
-  constructor() {
+  constructor(workers: WorkerUcpInfo[] = []) {
```

Comment:
I believe here we'll need the ability to control all the configuration options passed to `new Context()` below.